tools/python/m2k-osc/aout_tab.py

```diff
@@ -0,0 +1,242 @@
+import matplotlib
+import numpy as np
+import tkinter as tk
+from tkinter import ttk
+import matplotlib.figure as  fg
+import libm2k
+from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg, NavigationToolbar2Tk
```

Comment:
are all imports required?
or can some of them be reduced?
same question for globals

Reply:
imports and globals reduced